### PR TITLE
Productions

### PR DESCRIPTION
**Pull request by jepetersohn**

### CODE REVIEW

File: _layouts/production.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  <div class="production-posts">
+      <div class="production-post spacing">
+        <h1>{{ page.title }}</h1>
+          <b><span class="date">
+            {{ page.dates }} | {{page.location}}
+          </span></b>
+        </p>
+        {{ page.synopsis }}
+        <h3>PRODUCTION TEAM:</h3>
+        <h4>Director: <a href="{{page.director_bio_url}}">{{page.director}}</a>
```
Comment (TylerSmall19) [inline _layouts/production.html]: `<H4>` on L13 doesn't have an end tag.

File: _layouts/production.html [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        <h4>Cast:</h4>
+
+        {% for cast in page.cast %}
+          {% if cast.actor_bio_url %}
+            <a href="{{cast.actor_bio_url}}">{{cast.actor}}</a> as {{cast.role}}
```
Comment (TylerSmall19) [inline _layouts/production.html]: If these are in a list or a paragraph tag, you'd eliminate the `<br>`. I think this is a better way, semantically, to convey it's a list.

File: productions.html [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  <h2 class="spacing">{{page.title}}</h2>
+
+  <div class="production-posts">
+    {% for post in site.productions %}
+      <div class="production-post spacing">
+        <h3><a href="{{ post.url }}">{{ post.title }}</a></h3>
+          <span class="date">
+            {{ post.date | date: '%B %d, %Y' }}
+          </span>
+        </p>
```
Comment (TylerSmall19) [inline productions.html]: Where does this `</p>` tag start?

**Review by TylerSmall19 — CHANGES_REQUESTED** [review]
Hey, I left a few comments on the structure of the HTML and unclosed/unopened tags.

**Comment by cdfuller** [comment]
WORKIN ON THIS

File: _layouts/production.html (hunk 1 above)
Comment (jepetersohn) [reply _layouts/production.html]: all fixed

File: _layouts/production.html (hunk 2 above)
Comment (jepetersohn) [reply _layouts/production.html]: made them li's

**Review by TylerSmall19 — APPROVED** [review]

**Pull request merged**